docs/longevity.md

```diff
@@ -0,0 +1,41 @@
+# Longevity tests
+
+## Test operations
+
+On a high level overview, the test operations are:
+
+### Setup
+
+1) Instantiate a Cluster DB, with the specified number of nodes (the number
```

Comment:
`Instantiate` --> `Provision`

Reply:
that's equivalent, but for the sake of matching the code structure might make more sense.

Anyhow this whole section was cut of the README as is, longevity probably can be explien with more clear details then this old description.

Reply:
changed